docs/yaml/yaml.go

```diff
@@ -86,12 +107,15 @@ func GenYamlTreeCustom(cmd *cobra.Command, dir string, filePrepender func(string
 // GenYamlCustom creates custom yaml output
 // nolint: gocyclo
 func GenYamlCustom(cmd *cobra.Command, w io.Writer) error {
-	cliDoc := cmdDoc{}
-	cliDoc.Name = cmd.CommandPath()
+	cliDoc := cmdDoc{
+		Name:       cmd.CommandPath(),
+		Aliases:    strings.Join(cmd.Aliases, ", "),
+		Short:      forceMultiLine(cmd.Short, 73 /* = 80 - 'short: ' */),
```

Comment:
totally nit: could we extract those constants in a `const` block?

Reply:
I considered that, thought it would be more complicate to describe that `const` ("what is `short:` ?" "what is `long:`?" (they're the literal length of those strings in the yaml). So while it's a bit ugly, I thought it made more sense to describe it here inline.

Reply:
added some consts